src/assignment/pkg9/main.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment.pkg9;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+
+/**
+ *
+ * @author Jacob
+ */
+public class main {
+  public static void main(String args[]) throws IOException{
+   DiceFileReader reader = new DiceFileReader("dice.txt");
+   FileWriter writeFile = new FileWriter("diceresults.txt");
+   PrintWriter outputFile = new PrintWriter(writeFile);
+   
+   reader.read();
+   List<String> lines = new ArrayList<>();
```

Comment:
You don't need to make a new `ArrayList<>()` object here because you are going to replace it with `reader.getLines();` later

It would be fine to do
```
List<String> lines = reader.getLines();
```